cmd/loginStatus.go

```diff
@@ -51,7 +52,7 @@ func init() {
 			uotm := GetUserOAuthTokenManagerInstance()
 			tokenInfo, err := uotm.GetTokenInfo(ctx)
 
-			if err == nil && !tokenInfo.IsExpired() {
+			if err == nil && tokenInfo == nil {
```

Comment:
I'm a little lost here. How does tokenInfo being nil mean the token refreshed successfully?

Reply:
Token refresh is now seamlessly handled by the Az-identity codebase, alleviating concerns about token expiration.

Reply:
This doesnt quite answer the question. If the token is refreshed, why would it be nil?

Reply:
If AzIdentity is handling everything about the token, do we really need to even check token is there or not ?

Reply:
I think we might still want to check the expiry of the token since this is checking status of the login.